staging/src/kubevirt.io/application-aware-quota-api/pkg/apis/core/v1alpha1/types.go

```diff
@@ -145,7 +145,7 @@ type AAQSpec struct {
 	// PriorityClass of the AAQ control plane
 	PriorityClass *AAQPriorityClass `json:"priorityClass,omitempty"`
 	// namespaces where pods should be gated before scheduling
-	// Defaults to targeting namespaces with an "application-aware-quota/enable" label key.
+	// Defaults to targeting namespaces with an "application-aware-quota/enable-gating" label key.
 	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
```

Comment:
TBH, I'm not happy with this behavior in general as I think that the selector shouldn't determine which pods are gated, but rather which namespaces are affected by AAQ. This way if a namespace removes this label pods will be gated forever without an easy way to release them, except for removing the gates manually which IMO is at least a bad UX if not a bug.

Please refer to this issue for more information: https://github.com/kubevirt/application-aware-quota/issues/69.

However, the new name you suggested fits the current behavior, so I think it's valuable. In addition this API is alpha so we can always change it. So to conclude my concerns can be addressed in a follow-up.

/lgtm